ConcernsCaseWork/Service.TRAMS/Decision/DecisionDto.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Newtonsoft.Json;

```

Comment:
I'm not sure what this is, shouldn't there only be one DTO, and used by the concerns api ? 
(in the contracts proj btw)